jpo-ode-core/src/main/java/us/dot/its/jpo/ode/model/OdeAsn1Payload.java

```diff
@@ -1,42 +1,61 @@
 /*******************************************************************************
- * Copyright 2018 572682
- * 
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License.  You may obtain a copy
- * of the License at
- * 
- *   http://www.apache.org/licenses/LICENSE-2.0
- * 
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
- * License for the specific language governing permissions and limitations under
- * the License.
+ * Copyright 2018 572682 Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in
+ * writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
  ******************************************************************************/
+

```

Comment:
Updating classes that inherit OdeMsgPayload to now explicitly provide the OdeObject class as the type. This used to be the default, but had to change to allow for using ASN POJOs as part of the payload. Other check style changes were also applied.